modules/dnn/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ ocv_add_dispatched_file_force_all("int8layers/layers_common" AVX2 AVX512_SKX LAS
 ocv_add_dispatched_file_force_all("layers/cpu_kernels/conv_block" AVX AVX2)
 ocv_add_dispatched_file_force_all("layers/cpu_kernels/conv_depthwise" AVX AVX2 RVV LASX)
 ocv_add_dispatched_file_force_all("layers/cpu_kernels/conv_winograd_f63" AVX AVX2)
+ocv_add_dispatched_file_force_all("layers/cpu_kernels/fast_gemm_kernels" AVX AVX2)
```

Comment:
Do we have a comparison of GEMM vs. InnerProduct on AVX512, RVV and LASX?

Reply:
No. Optimization for AVX512, RVV and LASX will be the next patch I believe.

Reply:
Shall we make a switch to InnerProduct in such case at ONNX importer?
```cpp
#if CV_TRY_AVX512_SKX || CV_TRY_RVV || CV_TRY_LASX
// Create InnerProduct
#else
// Create GEMM
#endif
```

Reply:
Done. IIRC, Gemm introduced in this pr with AVX2 acceleration is even faster than the existing implementation with AVX512 acceleration. Let me collect the results if all tests are green.